src/tests/ftest/container/basic_snapshot.py

```diff
@@ -136,7 +136,7 @@ def test_basic_snapshot(self):
         Verify the snapshot is still available and the contents remain in
         their original state.
 
-        :avocado: tags=snapshot,basicsnap
+        :avocado: tags=snap,basicsnap
```

Comment:
Should _all_ be added here?

Reply:
This test needs to be removed. Waiting to investigate a little on it and then will remove it. All it's functionality has been taken care off under snapshot.py
So we can leave this test for now.